api/doc/thor.yaml

```
@@ -383,6 +383,33 @@ paths:
               schema:
                 $ref: '#/components/schemas/GetPeersResponse'
 
+  /node/metrics:
+    get:
+      tags:
+        - Node
+      summary: Retrieve Prometheus metrics
+      description: |
+        This endpoint allows clients to retrieve Prometheus metrics.
```

Can we leave a nod here that this endpoint is enabled conditionally? Also, does it throw a 404 if metrics are not enabled?

---

Something like this ?

```suggestion
        This endpoint allows clients to retrieve Prometheus metrics. Metrics are disabled by default and can be enabled using the CLI.
```

Yeah it's a 404 if the metrics are disabled.